test/built-ins/Temporal/PlainDate/prototype/toPlainDateTime/argument-string-time-designator-required-for-disambiguation.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindate.prototype.toplaindatetime
+description: ISO 8601 time designator "T" required in cases of ambiguity
+features: [Temporal, arrow-function]
+---*/
+
+const instance = new Temporal.PlainDate(2000, 5, 2);
+
+const ambiguousStrings = [
+  "2021-12",  // ambiguity between YYYY-MM and HHMM-UU
+  "1214",     // ambiguity between MMDD and HHMM
+  "0229",     //   ditto, including MMDD that doesn't occur every year
+  "1130",     //   ditto, including DD that doesn't occur in every month
+  "12-14",    // ambiguity between MM-DD and HH-UU
+  "202112",   // ambiguity between YYYYMM and HHMMSS
+];
+ambiguousStrings.forEach((arg) => {
+  assert.throws(
+    RangeError,
+    () => instance.toPlainDateTime(arg),
+    `${arg} is ambiguous and requires T prefix`
+  );
+  // The same string with a T prefix should not throw:
+  arg = `T${arg}`;
+  instance.toPlainDateTime(arg);
```

Comment:
Nit: I don't like reassigning function arguments much.
```suggestion
  instance.toPlainDateTime(`T${arg}`);
```